src/main/java/com/vmware/test/controller/CabServiceController.java

```diff
@@ -29,30 +28,24 @@
 	public String registerCab(@RequestBody Cab cab) {
 
 		boolean status = service.registerCab(cab);
-		if (status) {
-			logger.info("Registration Successful");
-			return "Registration Successful";
-		} else {
-			return "Registration unsuccessful";
-		}
+		return getString(status);
 	}
 
 	@PostMapping("/register/driver")
 	public String registerDriver(@RequestBody Driver driver) {
 
 		boolean status = service.registerDriver(driver);
-		if (status) {
-			logger.info("Registration Successful");
-			return "Registration Successful";
-		} else {
-			return "Registration unsuccessful";
-		}
+		return getString(status);
 	}
 
 	@PostMapping("/register/user")
 	public String registerUser(@RequestBody User user) {
 
 		boolean status = service.registerUser(user);
+		return getString(status);
+	}
+
+	private String getString(boolean status) {
```

Comment:
What is it for? is this method is being used anywhere else? if not please do not create a separate method as it will introduce method call overhead.

Reply:
okay, I agree. we will remove this method as it is not being used in any other place.